astropy/wcs/src/wcslib_wrap.c

```diff
@@ -3993,12 +3994,32 @@ PyWcsprm_set_zsource(
   return set_double("zsource", value, &self->x.zsource);
 }
 
+
+static PyObject*
+PyWcsprm_get_aux(
+    PyWcsprm* self,
+    /*@unused@*/ void* closure) {
+
+  PyObject* result;
+  double test = 1.2;
+
+  result = (PyObject *)PyAuxprm_cnew((PyObject *)self, self->x.aux);
+
+  if (result == NULL) {
+    Py_DECREF(result);
+    return NULL;
+  }
+
+  return result;
+}
+
 /***************************************************************************
  * PyWcsprm definition structures
  */
 
 static PyGetSetDef PyWcsprm_getset[] = {
   {"alt", (getter)PyWcsprm_get_alt, (setter)PyWcsprm_set_alt, (char *)doc_alt},
+  {"aux", (getter)PyWcsprm_get_aux, NULL, NULL},
```

Comment:
Add documentation here (last element) and in `docstrings.py`: https://github.com/astropy/astropy/blob/master/astropy/wcs/docstrings.py This will document `aux` member on https://docs.astropy.org/en/stable/api/astropy.wcs.Wcsprm.html

Reply:
Done